delivery_recipe/security_policy.md

```diff
@@ -0,0 +1,95 @@
+
+* A Senior Information Risk Owner (SIRO).
+* Information Asset Owner (IAO).
+* Other specialists relevant and specific to the organisation’s needs.
+
+The non hierarchical structure of the business means that all members of the company including the CEO are actively involved in the quest to maintain digital security and observe any findings that can improve our security strategy.
+
+##Culture and Awareness
+
+Our open and involved company culture enthuses a security forward ethos supported by all users and employees of the company. A broad spectrum of technical knowledge across the team provides a wide source of security knowledge and threat avoidance techniques. We have a “one team” ideal whereby everyone is responsible for the company as a whole and we work in coordination to maintain the environment.
+
+##Risk Management
+
+We work directly with digital services which inherently attract risks which need to be assessed by the team to make information and business enhancing decisions. We have:
+
+* A forward and informed understanding of the risks presented to the company
+* An accessible and clear collection of policies and procedures are implemented to support good risk management techniques
+* Well informed technical staff are involved in the analysis of threats, vulnerabilities and potential impacts associated with our activities
+
+##Technology and Services
+
+To comply with data protection laws and to remain resilient to cyber threats we will use the following implementations to make sure that security is integral to the development and deployment of services as part of usual working practises we will:
+
+* Observe advances in technology and identify any vulnerabilities or issues that may be applicable to customers development environments and undertake any necessary steps to reduce possible risk
+* Mitigate applicable threats
+* Ensure security controls are current and actively managed
+* Protect against, detect and correct malicious behaviour
+* Ensure that critical technology and services are resilient to disruptive challenges such as cyber attacks, and have the means to recover from these
+
+### Personnel Security
+Our employees are central to the maintenance of a good security structure and therefore the employment of these individuals is key to the safety of the company and its customers. To mitigate the issue of vulnerabilities caused by staff we will:
+
+* Employ individuals using the Baseline Personnel Security Standard for anyone who has access to any government assets
+* Put processes in place to evaluate levels of risk presented by an individual with appropriate levels of checks
+* Provide regular security appraisals and nurture a security conscious working environment
+
+### Physical Security
+Physical security measures will also be used to provide a secure and unhindered working environment for staff that protects against a range of threats.
+
```

Comment:
@steveparks rather than specifying any particular tools here I've added a more generic line:

* Ensure access to data remains secure through the use of digital tools and access controls

We can keep the actual details of how we secure and the tools we use in our management system which we don't have to publish. I don't want to give people any clues, only that it's all been considered.